roles/openshift_certificate_expiry/test/conftest.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+from OpenSSL import crypto
+
+# TODO: parameterized fixtures for valid/invalid certs
+# TODO: parameterized fixtures for different types of certs
+
+
+@pytest.fixture(scope='session')
+def ca_dir(tmpdir_factory):
+    return tmpdir_factory.mktemp('ca')
```

Comment:
Not at the session scope, and with the costly openssl operations it seemed better to only create them once per session.